hrjobcontract/CRM/Hrjobcontract/BAO/HRJobContract.php

```diff
@@ -193,21 +194,26 @@ protected static function getContractDates($contracts) {
     // If there are two (or more) Contracts starting on the same day
     // then we pick only the one with latest End date.
     foreach ($contracts['values'] as $contract) {
-      $details = civicrm_api3('HRJobDetails', 'getsingle', array(
+      $details = civicrm_api3('HRJobDetails', 'get', array(
```

Comment:
Problem is if you don't have a BAO method for something as basic as a `get`, it means that you need to manually implement it (and write the sql), just to get rid of the performance issues of the api...is that correct?